examples/radial_histogram_equalization.py

```diff
@@ -0,0 +1,140 @@
+"""
+===================================
+Radial Histogram Equalization
+===================================
+
+This example applies the radial histogram equalizing filter (`sunkit_image.radial.rhef`) filter to a sunpy map.
```

Comment:
I need to check if this works or if we need to add one more line to the API docs.